optimum/exporters/tasks.py

```diff
@@ -292,6 +292,46 @@ class TasksManager:
         "timm": "default-timm-config",
     }
 
+    _DIFFUSERS_SUPPORTED_MODEL_TYPE = {
+        "clip-text-model": supported_tasks_mapping(
+            "feature-extraction",
+            onnx="CLIPTextOnnxConfig",
+        ),
+        "clip-text-with-projection": supported_tasks_mapping(
+            "feature-extraction",
+            onnx="CLIPTextWithProjectionOnnxConfig",
+        ),
+        "unet": supported_tasks_mapping(
+            "semantic-segmentation",
+            onnx="UNetOnnxConfig",
+        ),
+        "vae-encoder": supported_tasks_mapping(
+            "semantic-segmentation",
+            onnx="VaeEncoderOnnxConfig",
+        ),
+        "vae-decoder": supported_tasks_mapping(
+            "semantic-segmentation",
+            onnx="VaeDecoderOnnxConfig",
+        ),
+    }
+
+    _TIMM_SUPPORTED_MODEL_TYPE = {
+        "default-timm-config": supported_tasks_mapping("image-classification", onnx="TimmDefaultOnnxConfig"),
+    }
+
+    _SENTENCE_TRANSFORMERS_SUPPORTED_MODEL_TYPE = {
+        "clip": supported_tasks_mapping(
+            "feature-extraction",
+            "sentence-similarity",
+            onnx="SentenceTransformersCLIPOnnxConfig",
+        ),
+        "transformer": supported_tasks_mapping(
```

Comment:
@JingyaHuang We don't override `model_type`, but simply add an attribute `export_model_type` to avoid to have multiple duplicate keys in this dictionary (or in timm dictionary).

Anyway, having `AutoConfig` in `modeling_base.py` is probably not a good thing.